R/client-cloud.R

```diff
@@ -86,6 +88,31 @@ cloudClient <- function(service, authInfo) {
 
         path <- paste0("/outputs/", output_id)
         output <- GET(service, authInfo, path)
+      } else if (deploymentRecordVersion == "unknown") {
```

Comment:
Adding this condition means the other conditions aren't strictly necessary. I'm on the fence about whether we should keep the DCF versioning and version handling logic.

Reply:
I feel like it's probably worth it, because it does save an API call, and we can eventually get the right logic into the IDE too.